3.0-RELEASE-NOTES.md

```diff
@@ -61,4 +61,16 @@ silently ignored. This has been fixed in 3.0, therefore an undefined mixin
 applied to a LoopBack model will throw an error that needs to be handled.
 
 See [related code change](https://github.com/strongloop/loopback-datasource-juggler/pull/944)
+for more details.
+
+##Remove deprecated model hooks and model events
+
+The following deprecated model events are now removed.
+
+Model events
+*changed
+*deletedAll
+*deleted
```

Comment:
Also please add instructions explaining how to upgrade code depending on these removed events (i.e. use Operation Hooks).